### DISCUSSION TITLE
Search within Squads & Resizable video windows

### DISCUSSION THREAD

**Discussion by richardhovdsveen** [dailydotdev/daily#1400] [Feature Requests]
Hi,

I often find myself missing two features that would greatly enhance my experience:

Search within Squads: Just like the "search in group" feature on Facebook, I would love the ability to search within a specific squad. Squad pages can become chaotic, and even if I remember where I found certain information, it often gets lost in the stream of activity. While the general search can find relevant matches, being able to search within a specific squad would make it much easier to locate the exact information I need.

Resizable video windows: When developers showcase examples, the screens often contain small text, along with an image of themselves and other details. While the fullscreen option is available, I don't always want to use it. It would be incredibly helpful to have the option to resize videos without going fullscreen, so I can adjust the view to better suit my needs.

Thank you for considering these suggestions!

**Comment by idoshamun** [comment]
Hi @richardhovdsveen,

Thanks a lot for the suggestions :)

As for the resizable videos, do you mean something similar to YT cinema mode? Or literally resizable like a textbox?

**Comment by richardhovdsveen** [comment]
Hi,

If it would be possible to size it up by to window-fullscreen that would be good. Then I could resize window to necessary size.


--
Med vennlig hilsen,
Richard Hovdsveen
Tlf. 959 27 685<tel:95927685>



Fra: Ido Shamun ***@***.***>
Dato: søndag, 8. september 2024 kl. 09:27
Til: dailydotdev/daily ***@***.***>
Kopi: Richard Hovdsveen ***@***.***>, Mention ***@***.***>
Emne: Re: [dailydotdev/daily] Search within Squads & Resizable video windows (Discussion #1400)

Hi @richardhovdsveen<https://github.com/richardhovdsveen>,

Thanks a lot for the suggestions :)

As for the resizable videos, do you mean something similar to YT cinema mode? Or literally resizable like a textbox?

**Comment by lucasdu4rte** [comment]
I was going to open a request about this, but I found this thread. 

To share my experience, after opening a feed with an embedded video, the video becomes small, forcing me to leave the page to view the video. 

Would it be possible to have a theatre mode to increase the size of the video player without making it full screen?

**Reply by sabdailydev** [reply]
@lucasdu4rte Thanks for the suggestion. We will consider adding the feature in the future, however it is not on the priority list yet.

**Comment by Debajyati** [comment]
I was going to open a new discussion but thought to check existing discussions and found this one.

It would be great and helpful if you people add a nice search bar for every single squad.

While the global search option is great, it would be a lot better if we get a search bar for searching posts specifically in a certain squad. Manually scrolling through a squad home page and look for a post you don't remember the whole title is time consuming and painful, specially when you are sure that the post was from the certain squad and from global search you can't find it among the vast matching article list.

I think, I have clearly explained, what is the feature and why is it important.

**Comment by sabdailydev** [comment]
@Debajyati Yes this is an important feature and it would be great to have it. However we don't have the bandwidth to fix this at the moment. Let me open it for community contribution in case someone is willing to build it.
